src/lib.rs

```diff
@@ -99,7 +99,7 @@ mod errors {
         errors {
             PyException(message: String) {
                 description("Python Exception")
-                display("Python Exception: {:?}", message)
+                display("Python Exception: {}", message)
```

Comment:
Wait, hold on. `message` is just a String here, this isn't the Rust error, is it?

This just handles the case of sending an exception from Python back to Python.

Reply:
yep I just thing that we're mixing a Debug fmt string it a field for display